ts/src/bitmart.ts

```diff
@@ -411,6 +411,32 @@ export default class bitmart extends Exchange {
                 'defaultNetworks': {
                     'USDT': 'ERC20',
                 },
+                'networksById': {
+                    'SOLANA': 'SPL',
```

Comment:
```suggestion
                    'SOLANA': 'SOL',
```

In the mapping file, you specified `SOL` and not `SPL`

Reply:
yes